cfn-resources/go.mod

```diff
@@ -36,6 +36,7 @@ require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/go-openapi/jsonpointer v0.19.6 // indirect
 	github.com/go-openapi/swag v0.22.4 // indirect
+	github.com/golang/mock v1.6.0
```

Comment:
this is unused and probably a consequence of doing `go install` for having the bin
when needing only the bin consider doing
```
go install github.com/golang/mock/mockgen@latest
```
to avoid polluting your go.mod

Reply:
I'm still going over this PR as I don't think you need mockgen yet

Reply:
may I ask you why is `unused`? I am using "github.com/golang/mock/gomock" in the `[...]_test.go` class

Reply:
oh you are right here, never mind me